src/Esprima/JavascriptParser.cs

```diff
@@ -4506,8 +4561,10 @@ private FunctionExpression ParseGeneratorMethod(bool isAsync = false)
 
         _context.AllowYield = true;
```

Comment:
Seems suspicious. Yield is enabled in the parameter list just because it's a generator method? Shouldn't it be enabled for the method body only?